Fixed login form, added twitter field. Reduced size of both forms

lili2311 opened the pull request:
###### @rimpey
### [PIMP-220](https://womenhackfornonprofits.atlassian.net/browse/PIMP-220)
- Added twitter handle
- reduced size of login and register forms
- structured login form better
- made post code and city fields smaller as they are too big for the values

![screen shot 2016-03-06 at 01 53 03](https://cloud.githubusercontent.com/assets/2911613/13551798/71d7130e-e33e-11e5-9792-4a772379deb0.png)
![screen shot 2016-03-06 at 01 53 19](https://cloud.githubusercontent.com/assets/2911613/13551800/781ec694-e33e-11e5-9468-96b7dd0b2d0e.png)

tanyapowell committed:
sample config file added

Commit:
added triangles to media section and added button width to forms to be the same as nav button

Commit:
Fixed login form, added twitter field. Reduced size of both forms

Commit:
Fixed login form, added twitter field. Reduced size of both forms

Commit:
Merge pull request #19 from womenhackfornonprofits/PIMP-217

added triangles to media section and added button width to forms to b…

rimpey commented:
Feedback for Login Only:
Changes for Pages

1) main.scss => swap order of imports just to make sure styles extended correctly

```
@import "pages/home";
@import "pages/contact";
```

2) index.html => just change class 'column' to 'columns' and include row classes - just realised now needed to be in rows to create correct spacing - update for social media

```
        <!-- **************** -->
        <!-- TALK ABOUT PANEL -->
        <!-- **************** -->

        <div id="pnl-talkabout-colr-to-edge">
            <div class="row pnl-talkabout">
                
                <!-- title -->
                <div class="row">
                    <div class="large-12 columns"> 
                        <div class="pnl-title">
                            <h1>What we're talking about</h1>
                        </div>
                    </div>
                </div> <!-- end row for title -->

                <!-- social media boxes -->
                <div class="row">
                    <div class="large-6 columns"> 
                        <div class="social-tile">
                            <h4 class="social-tile-title">Latest News</h4>
                        </div>                           
                    </div>
                    <div class="large-6 columns">
                        <div class="social-tile"> 
                            <h4 class="social-tile-title">Latest Tweet</h4>
                        </div>                           
                    </div>
                </div> <!-- end row for social media boxes -->
                
            </div> <!-- end row pnl-talkabout -->
        </div>
```

 3) login.html

```
<!DOCTYPE html>
<html class="no-js" lang="en">
    
    <!--#include virtual="includes/head.html" -->
    
    <body>
        <!--#include virtual="includes/header.html" -->
        
            <div class="pnl-title pnl-login">
            <h1>Log In</h1>

            <form id="sign-in" method="POST" action="">
                <div class="row">
                     <div class="large-6 large-centered columns">
                         <label for="email">Email:</label>
                         <input type="email" name="email" placeholder="Username" required>
                     </div>
                </div>
                <div class="row">
                    <div class="large-6 large-centered columns">
                         <label for="password">Password:</label>
                         <input type="password" name="password" placeholder="Password">
                         <a href="#">Forgot Password</a>
                    </div>
                </div>
                <div class="row">
                    <div class="large-6 large-centered columns">
                        <input class="button right" type="submit" value="Sign in">
                        <a href="create-account.html" class="underline txt-right">Not Registered? Start here</a>
                    </div>
                </div>
             </form>
           
        </div> <!-- end pnl-title pnl-login -->
        
        <!--#include virtual="includes/footer.html" -->
        
        <!--#include virtual="includes/mobile-menu-end.html" -->

        <!--#include virtual="includes/foot.html" -->

    </body>
</html>
```

4) Styling for login.scss

```
/* =================================
   PAGES / LOGIN 
   ================================= */

```
```
.pnl-login {
    padding-top: rem-calc(65); 
    padding-bottom: rem-calc(40);
}

.pnl-login {
    
    @extend .pnl-login; 
    @extend .pnl-dark-txt;
    background-color: $white;
}
```

rimpey commented:
Comments for the Registration form to follow...

rimpey commented:
.. sorry correction for frontend/src/styles/pages/_login.scss

```
/* =================================
   PAGES / LOGIN 
   ================================= */

.pnl-login {
    @extend .pnl-dark-txt;
    padding-top: rem-calc(65); 
    padding-bottom: rem-calc(40);
    background-color: $white;
}
```

rimpey committed:
Added the Featured Causes Panel - used slick slider included new fonts directory added new images for slider content with slider styling

rimpey committed:
Modified path for gh-pages

Commit:
added seeder.js

Mstot committed:
corrected the name of the database to match the project

tanyapowell committed:
config.js removed

Mstot committed:
Updated README with info about seeder file + corrected how to run locally

tanyapowell committed:
Merge pull request #22 from womenhackfornonprofits/config-change

Config change

Mstot committed:
corrected a comment

Mstot committed:
Merge pull request #23 from womenhackfornonprofits/PIMP-229

PIMP-229

rimpey committed:
Foundation equalizer classes on slider columns to maintain same height columns
Included Sliders for both Causes and Marketers
Amended Summary tiles to include 'Read Me' link
Added styling for Social Media panel - Latest News and Twitter Widget

rimpey committed:
Merge pull request #21 from womenhackfornonprofits/pimp-74

Added the Featured Causes Panel - used slick slider included new font…

rimpey committed:
Modified Login and Registration pages to use Foundation classes

rimpey committed:
Modified Login and Registration to use Foundation classes

Merge branch 'PIMP-220' of https://github.com/womenhackfornonprofits/pimpmycause-rebuild into PIMP-220
